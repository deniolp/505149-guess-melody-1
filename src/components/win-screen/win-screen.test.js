import React from 'react';
import renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';

import WinScreen from '../win-screen/win-screen';

describe(`WelcomeScreen`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(<BrowserRouter>
      <WinScreen
        mistakes={0}
        onReplayButtonClick = {jest.fn()}
      />
    </BrowserRouter>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
