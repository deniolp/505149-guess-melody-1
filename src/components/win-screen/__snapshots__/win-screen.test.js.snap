// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomeScreen renders correctly 1`] = `
<section
  className="result"
>
  <div
    className="result__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    className="result__title"
  >
    Вы настоящий меломан!
  </h2>
  <p
    className="result__total"
  >
    Вы выиграли, , совершив 
    0
     ошибки(-у, -ок)
  </p>
  <a
    className="replay"
    href="/"
    onClick={[Function]}
  >
    Попробовать ещё раз
  </a>
</section>
`;
